### ISSUE TITLE
Import api throw error, when request not setting timestamps

### ISSUE THREAD

**Issue by zhyhang** [pilosa/pilosa#1813]
- V1.2
- **Import api /index/%s/field/%s/import throw: "runtime error: index out of range", when request not setting timestamps.**
- Is it new rule: must set timestamps in request in V1.2?
- But in V1.1, that not be required.

**Comment by yuce** [comment]
@zhyhang Thanks for reporting this issue. What's the version of the Go client library are you using ?

**Comment by zhyhang** [comment]
@yuce java-client-pilosa 1.0.0

**Comment by yuce** [comment]
I thought you were using the Go client but it seems you're using the Java client. Are you using the latest 1.2.0 of the Java client ?

**Comment by yuce** [comment]
Java Client 1.0.0 is not compatible with Pilosa 1.2.0. Could you upgrade to 1.2.0 ?

**Comment by zhyhang** [comment]
Version 1.2.0 removes the importRowCols() method, so I must modify my codes. But I think the server 's api.go line 860 possiblly arbitrary.

```
			for i, colID := range req.ColumnIDs {
				shard := colID / ShardWidth
				if _, ok := m[shard]; !ok {
					m[shard] = make([]Bit, 0)
				}
				m[shard] = append(m[shard], Bit{
					RowID:     req.RowIDs[i],
					ColumnID:  colID,
					Timestamp: req.Timestamps[i],
				})
			}
```

**Comment by yuce** [comment]
I don't think we had importRowCols` method for the Java client before, but if you are able to paste the code which breaks with Java client 1.2, I can suggest a replacement.

**Comment by zhyhang** [comment]
@yuce Thanks! upgrade to 1.2 is OK.
